server/repository/migrations/postgres/2023-01-25T10-00_create_clinician_table/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE clinician
+(
+    id TEXT NOT NULL PRIMARY KEY,
+    store_id TEXT NOT NULL REFERENCES store(id),
+    code TEXT NOT NULL,
+    last_name TEXT NOT NULL,
+    initials TEXT NOT NULL,
+    first_name TEXT,
+    registration_code TEXT,
+    category TEXT,
+    address1 TEXT,
+    address2 TEXT,
+    phone TEXT,
+    mobile TEXT,
+    email TEXT,
+    female BOOLEAN NOT NULL,
```

Comment:
Just noticed, this should be is_female, right? (its just `female` in mSupply)

Reply:
`is_female` on om is more consistent